include/openssl/objects.h

```diff
@@ -642,6 +642,16 @@
 #  define NID_ripemd160WithRSA            119
 #  define OBJ_ripemd160WithRSA            1L,3L,36L,3L,3L,1L,2L
 
+#  define SN_blake2b                      "BLAKE2b"
+#  define LN_blake2b                      "blake2b"
+#  define NID_blake2b                     1022
+#  define OBJ_blake2b                     1,3,6,1,4,1,1722,12,2,1,16
+
+#  define SN_blake2s                      "BLAKE2s"
+#  define LN_blake2s                      "blake2"
```

Comment:
On Wed, Mar 9, 2016 at 3:42 PM, Kurt Roeckx <notifications@github.com>
wrote:

> In include/openssl/objects.h
> <https://github.com/openssl/openssl/pull/566#discussion_r55610855>:
>
> > @@ -642,6 +642,16 @@
> >  #  define NID_ripemd160WithRSA            119
> >  #  define OBJ_ripemd160WithRSA            1L,3L,36L,3L,3L,1L,2L
> >
> > +#  define SN_blake2b                      "BLAKE2b"
> > +#  define LN_blake2b                      "blake2b"
> > +#  define NID_blake2b                     1022
> > +#  define OBJ_blake2b                     1,3,6,1,4,1,1722,12,2,1,16
> > +
> > +#  define SN_blake2s                      "BLAKE2s"
> > +#  define LN_blake2s                      "blake2"
>
> I'm not sure this is auto generated? At least make update doesn't seem to
> re-add it.
>
I think I wrote those lines by hand when trying to figure out the object
system in OpenSSL.  I removed them in my local version, and it seems to
work without them.  I don't think objects.h gets generated anymore.